pkg/test/expectations/expectations.go

```diff
@@ -105,7 +105,7 @@ func ExpectCreatedWithStatus(c client.Client, objects ...client.Object) {
 	}
 }
 
-func ExpectDeleted(c client.Client, objects ...client.Object) {
+func ExpectDeleted(ctx context.Context, c client.Client, objects ...client.Object) {
```

Comment:
I must be missing something here - why did you add ctx here? You don't seem to use it AFAICT